cypress/support/commands/commands.ts

```diff
@@ -59,3 +59,12 @@ Cypress.Commands.add('iFrame', () => {
     .should('not.be.empty')
     .then((body) => cy.wrap(body));
 });
+
+/**
+ * Create resource name
+ */
+Cypress.Commands.add('createE2EResourceName', (context) => {
+  const runTimestamp = +new Date();
```

Comment:
This needs to live outside of the function, otherwise it will be new every time it's called